.husky/pre-commit

```diff
@@ -0,0 +1,2 @@
+npx pretty-quick --staged
+npm run lint
```

Comment:
Note: Looks redundant at first glance, when `pretty-quick` formats the code already. But the linter can still fail your code (in CI) if you didn't lint before committing.